src/main/java/no/digipost/api/client/DigipostClient.java

```diff
@@ -71,11 +73,12 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(DigipostClient.class);
 
-    private final EventLogger eventLogger;
-    private final ApiService apiService;
-    private final MessageSender messageSender;
-    private final MessageDeliverer deliverer;
-    private final DocumentCommunicator documentCommunicator;
+	private final EventLogger eventLogger;
```

Comment:
Jeg liker veldig godt at GitHub bruker _tab width=8 spaces_, for da blir det veldig tydelig hvor man har ukonsistent bruk av whitespace i samme fil ;)